gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.15.4
+version=30.15.5
```

Comment:
This should be 30.0.0

Doing a major version would also mean waiting to merge some other backwards incompatible changes so they can all be bundled in on major version. 
It would add additional time to the release, let us know if that is okay with you.

Reply:
It's not urgent to fail invalid parameters. We can do as you suggested and split the change.